docs/core/howto/testing.rst

```diff
@@ -246,3 +246,7 @@ then deletes that schema in the tearDown function, your tests will behave in an
 unpredictable fashion as they tromp upon each other if they have their own
 schema.  And this won't actually indicate a real error in your code, merely a
 testing-specific race-condition.
+
+Trial provides `TWISTED_TRIAL_PARALLEL_INDEX` environment variable to the tests when run in parallel.
```

Comment:
I think that this documentation is good enough.

Just a curiosity and for future reference,

Do you know how other  testing frameworks are dealing with this issue?

Do they also set an env variable?
Is there some convention for the variable name?

Are there any other things that can help with shared resources?